### ISSUE TITLE
/? not working

### ISSUE THREAD

**Issue by HoldYourWaffle** [ritchielawrence/cmdow#10]
`cmdow.exe /?` just outputs:
`Error: Unrecognized argument(s). Use CMDOW /? for help.`

**Comment by ritchielawrence** [comment]
At an absolute minimum, please include EXACT OS version, version of cmdow.exe, checksum of cmdow.exe, where you got cmdow.exe from, how you got it to your computer and the EXACT steps to reproduce the issue. -- Thanks.

**Comment by HoldYourWaffle** [comment]
Ah shit forgot about that. Sorry my bad 😅 

I discovered that it only breaks when using Cygwin terminal.

OS: Windows 10 1709 16299.15 (exact enough?)
Version of cmdow: 1.4.8
Where I got it from: it doesn't seem to matter, I tried github releases & the [download zip](http://ritchielawrence.github.io/cmdow/) option.
How did I get it: chrome download
Checksums:
- MD5: `ddd12566b99343b96609afa2524ecec3`
- SHA1: `8fef2c2bc87ef7d135296fdb4cf9ecd9c0322d55`
- SHA256: `767b877e735c425bf05c34683356abfde4070b092f17a4741ea5ac490611f3de`

Exact steps:
- Open a cygwin terminal
- Execute the `./cmdow.exe /?` command in the directory you downloaded it.
